Solutions/Corvus.Monitoring.ApplicationInsights/Corvus/Monitoring/ApplicationInsights/AiOperationsInstrumentation.cs

```diff
@@ -28,7 +29,7 @@ public AiOperationsInstrumentation(TelemetryClient telemetryClient)
             this.telemetryClient = telemetryClient;
         }
 
-        /// <inheritdoc />
+                /// <inheritdoc />
```

Comment:
This indentation change looks weird in GitHub's diff view. Does it look OK locally? And if so, there isn't some mixture of tabs and spaces here is there?